frontend/src/components/navigation/Header.css

```diff
@@ -0,0 +1,150 @@
+.NavbarItems {
+  /*background: linear-gradient(90deg, rgb(110, 94, 254) 0%, rgba(73,63,252,1) 100%);*/
+  background: linear-gradient(90deg, rgb(189, 186, 218) 0%, rgb(205, 201, 238) 100%);
+  height: 80px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 1.2rem;
+}
+
+.navbar-logo {
+  color: #fff;
+  justify-self: start;
+  margin-left: 20px;
+  cursor: pointer;
+}
+
+/*.fa-react {
+  margin-left: 0.5rem;
+  font-size: 1.6rem;
+}*/
```

Comment:
There are a number of commented out components throughout the codebase. As a general rule, when you finalize a PR and are ready to merge your code, you want to be sure that all old, unused, commented-out code is removed.

There are a fair number of other places where you have commented-out code -- be sure to either remove or keep all of these, depending on whether they're being used or not.

To be clear, this means you won't need this file any more, but the code styling (not the classes, but the styling) will be relevant 👍